src/screens/IndexScreen.js

```diff
@@ -253,6 +253,7 @@ export const IndexScreen = ({ navigation, route }) => {
                 ListEmptyComponent={<EmptyMessage title={texts.empty.list} />}
                 navigation={navigation}
                 data={listItems}
+                noEvent={listItems.length > 0 ? false : true}
```

Comment:
i think this new prop can be avoided and does not need to be passed down. at the end there is `ListHeaderComponent={noEvent ? null : ListHeaderComponent}` now. so i have in my mind to directly avoid passing something as `ListHeaderComponent` here if there is no data instead of a new prop to compare later, you know?

Reply:
Yes I understand that I don't need the extra prop.